content/docs/pipelines/sdk/static-type-checking.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "DSL Static Type Checking"
+description = "Statically check the component I/O types"
+weight = 6
++++
+
+## Overview
```

Comment:
For some reason the formatting isn't working here. See preview: https://deploy-preview-562--competent-brattain-de2d6d.netlify.com/docs/pipelines/sdk/static-type-checking/

Try adding a blank line after the heading.

Reply:
OK. I added blank lines around all the subtitles that start with '#'.